changelogs/unreleased/gh-8576-set-compat-options-to-new.md

```diff
@@ -0,0 +1,11 @@
+## feature/compat
```

Comment:
> Due to the approaching tarantool 3.0 release, the following compat options' defaults were updated to new behavior:

> * The following compatibility options' defaults were switched to new behavior:

There seems to be an extra apostrophe in the commit message and the changelog entry.

Reply:
It is intentional, because `options` is plural, see [Rule 2a](https://www.grammarbook.com/punctuation/apostro.asp).